src/main/java/org/dataloader/impl/NoOpValueCache.java

```diff
@@ -20,37 +22,55 @@
 @Internal
```

Comment:
Maybe promote it to public? We expose a `public static final NOOP` which may be of use to library consumers

Reply:
The NOOP is a default impl for never caching.  I want it to stay that way.  There is no real useful code here honestly for library consumers